hoodie-client/src/test/java/HoodieClientExample.java

```diff
@@ -84,7 +84,7 @@ public void run() throws Exception {
         Path path = new Path(tablePath);
         FileSystem fs = FSUtils.getFs();
         if (!fs.exists(path)) {
-            HoodieTableMetaClient.initTableType(fs, tablePath, HoodieTableType.valueOf(tableType), tableName);
+            HoodieTableMetaClient.initTableType(fs, tablePath, HoodieTableType.valueOf(tableType), tableName, "HoodieAvroPayload");
```

Comment:
should nt this be a fully qualified class name? Not sure if this works..

Reply:
This example is not used anywhere, I'll add the fully qualified name from the class, just put this string as a place holder in the meantime.